torch/ao/quantization/_pt2e/graph_utils.py

```diff
@@ -20,15 +20,23 @@
 ]
 
 
-def _create_equivalent_types_dict():
+def get_equivalent_types() -> List[Set]:
```

Comment:
You only need this. and maybe `update_equivalent_types`

Reply:
Great suggestion, it makes much less change to original code.